OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitYoutubeVideoFragment.java

```diff
@@ -0,0 +1,195 @@
+package org.edx.mobile.view;
+
+import android.content.res.Configuration;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+
+
+import org.edx.mobile.R;
+import org.edx.mobile.model.api.TranscriptModel;
+import org.edx.mobile.model.course.VideoBlockModel;
+import org.edx.mobile.player.TranscriptManager;
+
+
+import com.google.android.youtube.player.YouTubePlayer;
+import com.google.android.youtube.player.YouTubePlayer.Provider;
+import com.google.android.youtube.player.YouTubeInitializationResult;
+import com.google.android.youtube.player.YouTubePlayerSupportFragment;
+
+import java.io.InputStream;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+
+import subtitleFile.Caption;
+import subtitleFile.FormatSRT;
+import subtitleFile.TimedTextObject;
+
+public class CourseUnitYoutubeVideoFragment extends CourseUnitVideoFragment implements YouTubePlayer.OnInitializedListener {
+
+    private static final int SUBTITLES_DISPLAY_DELAY_MS = 100;
```

Comment:
Would it be better to make these three private variables protected in the superclass instead of redeclaring them here?

Reply:
The superclass doesn't have this variable but I can do this for HAS_NEXT_UNIT_ID and HAS_PREV_UNIT_ID